openspec/changes/add-application-control/tasks.md

```
@@ -0,0 +1,155 @@
+## 1. Demolition (single PR, delete the legacy scaffolding)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)